client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -156,7 +156,14 @@ define( [ 'utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view
                                 // queue model request
                                 self.deferred.reset();
                                 self.deferred.execute( function( process ) {
-                                    self._buildModel( process, { id : id, version : version } )
+                                    if ( options.hasOwnProperty( "workflow" ) ) {
```

Comment:
This should be moved out of the base class. I can help with this tho. Is this PR functional?